### PR TITLE
F favor map improve

### PR DESCRIPTION
**Pull request by basbeu**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/favors/favors/FavorCreateFragment.java [hunk 1]
```diff
@@ -184,7 +189,24 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
...
+            switch (message.what) {
+                case 1:
+                    Bundle bundle = message.getData();
+                    favorLocation = new GeoPoint(bundle.getDouble("latitude"), bundle.getDouble("longitude"));
+                    locationCityValid.set(true);
+                    break;
+                case 2:
+                    locationCityValid.set(false);
+                    break;
+                default:
```
Comment (alexander-san) [inline app/src/main/java/ch/epfl/sweng/favors/favors/FavorCreateFragment.java]: Can I add a return or break here?

File: app/src/main/java/ch/epfl/sweng/favors/favors/FavorDetailView.java [hunk 2]
```diff
@@ -146,7 +148,7 @@ public static void setImageUri(ImageView view, String imageName) {
         if (imageName == null) {
             view.setImageURI(null);
         } else {
-            view.setImageURI(Uri.parse("android.resource://ch.epfl.sweng.favors/drawable/"+imageName.toLowerCase().replaceAll("\\s","")));
+            view.setImageURI(Uri.parse(getIconPath(imageName)));
```
Comment (alexander-san) [inline app/src/main/java/ch/epfl/sweng/favors/favors/FavorDetailView.java]: nice change

File: app/src/main/java/ch/epfl/sweng/favors/database/User.java [hunk 3]
```diff
@@ -59,16 +60,11 @@ static public void resetMain() {
         user.reset();
     }
 
-    /**
-     * Sets the current location of the user
-     * @param geo the geopoint that will be stored in the database
```
Comment (alexander-san) [inline app/src/main/java/ch/epfl/sweng/favors/database/User.java]: can we keep the comments or were they wrong (not important rn though)

**Comment by alexander-san** [comment]
ch.epfl.sweng.favors.favors.FavorsCreateFragmentTest > createFavorFailsWhen0Tokens[test(AVD) - 5.1.1] [31mFAILED

ch.epfl.sweng.favors.main.CompleteLoggedUITest > completeLoggedUITest[test(AVD) - 5.1.1] [31mFAILED

**Comment by alexander-san** [comment]
Rerunning Travis

**Comment by nicolasp9-dev** [comment]
Useless to rerun Travis, test wasn't modified in order to pass with new functionalities atm, and it was known that this functionalities will make some tests fail, fix will be done after today's meeting

**Comment by alexander-san** [comment]
Ok, I was not sure if it was that or it was really broken, because I saw at least one which seemed to be failing unrelated.
I was just trying to help

**Pull request merged**